content/docs/composition-vs-inheritance.md

```diff
@@ -78,15 +78,16 @@ function App() {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/gwZOJp?editors=0010)
+[**Δοκιμάστε το στο CodePen**](https://codepen.io/gaearon/pen/gwZOJp?editors=0010)
 
-React elements like `<Contacts />` and `<Chat />` are just objects, so you can pass them as props like any other data. This approach may remind you of "slots" in other libraries but there are no limitations on what you can pass as props in React.
+Τα React elements όπως `<Επαφές />` και `<Chat />` είναι απλά objects, επομένως μπορείτε να τα περάσετε ως props όπως οποιαδήποτε άλλα δεδομένα. Αυτή η προσέγγιση μπορεί να σας υπενθυμίσει τα "slots" σε άλλες βιβλιοθήκες, αλλά δεν υπάρχουν περιορισμοί σε αυτό που μπορείτε να περάσετε ως props στο React.
 
-## Specialization {#specialization}
+## Ειδίκευση {#specialization}
 
-Sometimes we think about components as being "special cases" of other components. For example, we might say that a `WelcomeDialog` is a special case of `Dialog`.
+Μερικές φορές σκεφτόμαστε τα components ως "ειδικές περιπτώσεις" άλλων components. Για παράδειγμα, θα μπορούσαμε να πούμε ότι ένα `WelcomeDialog` είναι μια ειδική περίπτωση του `Dialog`.
+
```

Comment:
I think there is an extra line in this block. Imo It would be better if we have parity with the english file

Reply:
True!

Reply:
After having a look, the new line is needed. If you see the diff of the commits, the English translation also has space in between. It's just GitHub that issues this new line for some odd reason.